src/test/java/uk/gov/hmcts/ccd/v2/external/controller/CaseControllerTestIT.java

```diff
@@ -798,4 +798,106 @@ private SupplementaryDataUpdateRequest createSupplementaryDataIncrementRequest()
         return new SupplementaryDataUpdateRequest(requestData);
     }
 
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_cases.sql"})
+    void testShouldGetLinkedCases() throws Exception {
+        final String caseId = "1504259907353529";
+        final String URL = "/getLinkedCases/" + caseId;
+        mapper.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+        final MvcResult mvcResult = mockMvc.perform(get(URL)
+                .header(REQUEST_ID, REQUEST_ID_VALUE)
+                .contentType(JSON_CONTENT_TYPE))
+            .andReturn();
+
+        org.assertj.core.api.Assertions.assertThat(mvcResult.getResponse())
+            .isNotNull()
+            .satisfies(response -> org.assertj.core.api.Assertions.assertThat(response.getStatus()).isEqualTo(200));
+    }
+
+    @Test
+    @Sql(executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD, scripts = {"classpath:sql/insert_cases.sql"})
+    void testShouldGetLinkedCasesOptionalParameters() throws Exception {
+        final String caseId = "1504259907353529";
+        final String URL = "/getLinkedCases/" + caseId + "?startRecordNumber=2&?maxReturnRecordCount=1";
```

Comment:
The parameters here are wrong, don't need the second `?` before `maxReturnRecordCount`

Reply:
done

Reply:
Pushed Change to RDM_13138 branch https://github.com/hmcts/ccd-data-store-api/commit/4cabb90d98b54e323f0495166a5bf3dc407e9eda